packages/yew-macro/src/html_tree/mod.rs

```diff
@@ -321,3 +322,7 @@ impl ToTokens for HtmlRootBraced {
         });
     }
 }
+
+pub(crate) fn is_ide_completion() -> bool {
+    std::env::var_os("RUST_IDE_PROC_MACRO_COMPLETION").as_deref() == Some("1".as_ref())
+}
```

Comment:
Can you please move this to the root of the crate? It may be needed to be used outside of the `html_tree` module, so it's best to keep it at the root. There are other utils in the `lib.rs` file, and this function can go there as well

Reply:
Done!

Also I decided to use `RUST_IDE_PROC_MACRO_COMPLETION_DUMMY_IDENTIFIER` environment variable instead of `RUST_IDE_PROC_MACRO_COMPLETION` as a result of this discussion  https://github.com/rust-lang/rust-analyzer/issues/13731